### ISSUE TITLE
Fix Compiler Warnings

### ISSUE THREAD

**Issue by AgitoReiKen** [AgitoReiKen/ue-restyle#18]
User reported following compiler log when building project with Restyle
```
Warning        Modules must specify an explicit precompiled header (
eg. PrivatePCHHeaderFile = "Public/Restyle.h") from UE 4.21 

Warning        Library Restyle/Binaries/ThirdParty/MinHook/Win64/minhook.x64.lib' 
was not resolvable to a file when used in Module 'Restyle', 
assuming it is a filename and will search library paths for it. 
This is slow and dependency checking will not work for it. 
Please update reference to be fully qualified alternatively use PublicSystemLibraryPaths 
if you do intended to use this slow path to suppress this warning.   

Warning       Modules must specify an explicit precompiled header (
eg. PrivatePCHHeaderFile = "Public/RestyleShadersModule.h") from UE 4.21
```

**Comment by AgitoReiKen** [comment]
enabled pch
https://github.com/AgitoReiKen/ue-restyle/commit/23f5b0f9a409d9d1845c634218381050f1498ae9
https://github.com/AgitoReiKen/ue-restyle/commit/99264c31417f2a4ea1dfedc322162204855b54b3
